sw/device/lib/testing/mock_csr.h

```diff
@@ -0,0 +1,71 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_MOCK_CSR_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_MOCK_CSR_H_
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "sw/device/lib/base/csr.h"
+#include "sw/device/lib/testing/mask_rom_test.h"
+
+namespace internal {
```

Comment:
I think you forgot an outer namespace like `mock_csr`?

Reply:
Done